usecase/usecase-webapi-xwalk-tests/samples/Fingerprint/README.md

```diff
@@ -1,9 +0,0 @@
-# Fingerprint authentication feature example
-## Introduction
-This example is for testing the fingerprint authentication feature.
-## Prerequisites
-* Copy the index.html from [crosswalk-samples fingerpint sample](https://github.com/crosswalk-project/crosswalk-samples/fingerprint/www/index.html) to ```fingerprint``` directory.
```

Comment:
Please move this precondition to README.md file under usecase-webapi-xwalk-tests